### PR TITLE
support thumbnail section from video on edits

### PR DESCRIPTION
**Pull request by zxawry**
## PR Checklist

Please check all that apply to this PR using "x":

- [ ] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: #2531

## What is the current behavior?
video needs to be selected in order to take a new snapshot

## What is the new behavior?
video snapshot is enabled without selecting the video file

## Other information
I tackled this once with #2543 which turned out to be troublesome (sorry about the trouble) I think this one is quite the fix.

### CODE REVIEW

**Comment by tzarebczan** [comment]
Thanks for the PR @zxawry ! We'll get a review on this soon.

**Comment by seanyesmunt** [comment]
Thanks @zxawry 🙂 

We'll get this in after the upcoming app release

**Comment by zxawry** [comment]
Sure thanks! Let me know if I should make changes.

File: src/ui/component/selectThumbnail/index.js [hunk 1]
```diff
@@ -25,6 +26,7 @@ const select = state => ({
   myClaimForUri: selectMyClaimForUri(state),
   // If I clicked the "edit" button, have I changed the uri?
   // Need this to make it easier to find the source on previously published content
+  fileInfos: selectFileInfosByOutpoint(state),
```
[the remark this answers is not in the document]
Comment (zxawry) [reply src/ui/component/selectThumbnail/index.js]: Done! I didn't notice that, sorry!

**Pull request merged**